patches/did-jwt+5.12.4.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/did-jwt/lib/index.module.js b/node_modules/did-jwt/lib/index.module.js
```

Comment:
This file describes the change. removing this file is enough to remove the patch